rest-api/tools/ehr_upload.py

```diff
@@ -71,7 +84,7 @@ def get_participants_without_site_pairing(session, organization, source_bucket,
   parser.add_argument('--organization', help='The organization to find participants and sites for',
                       required=True)
   parser.add_argument('--source_bucket', help='The bucket to read from in one env.'
-                      ' i.e. ptc-uploads-pmi-drc-api-prod', required=True)
+                      ' i.e. ptc-uploads-pmi-drc-api-sandbox', required=True)
   parser.add_argument('--destination_bucket', help='The bucket to write to in one env.'
                       'i.e. ptc-uploads-pmi-drc-api-prod', required=True)
```

Comment:
The example values here should probably be for the same environment right?

Reply:
sure